pkg/notify/notifier/telegram/telegram.go

```diff
@@ -0,0 +1,179 @@
+package telegram
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kubesphere/notification-manager/pkg/async"
+	"github.com/kubesphere/notification-manager/pkg/controller"
+	"github.com/kubesphere/notification-manager/pkg/internal"
+	"github.com/kubesphere/notification-manager/pkg/internal/telegram"
+	"github.com/kubesphere/notification-manager/pkg/notify/notifier"
+	"github.com/kubesphere/notification-manager/pkg/template"
+	"github.com/kubesphere/notification-manager/pkg/utils"
+)
+
+const (
+	DefaultSendTimeout = time.Second * 3
+	URL                = "https://api.telegram.org/bot%s/sendMessage"
```

Comment:
Does it miss a ":" between `bot` and `token`?

<img width="623" alt="1683770174567" src="https://github.com/kubesphere/notification-manager/assets/53003665/51fe4ab4-cc31-428a-9085-c3566c90bb9c">

Reply:
the token is `123456:ABC-DEF1234ghIkl-zyx57W2v1u123ew11`